coverage/coverage.json

```diff
@@ -0,0 +1 @@
":39,"column":35}},{"start":{"line":39,"column":39},"end":{"line":39,"column":58}}]},"3":{"line":40,"type":"if","locations":[{"start":{"line":40,"column":4},"end":{"line":40,"column":4}},{"start":{"line":40,"column":4},"end":{"line":40,"column":4}}]},"4":{"line":50,"type":"if","locations":[{"start":{"line":50,"column":2},"end":{"line":50,"column":2}},{"start":{"line":50,"column":2},"end":{"line":50,"column":2}}]},"5":{"line":58,"type":"if","locations":[{"start":{"line":58,"column":2},"end":{"line":58,"column":2}},{"start":{"line":58,"column":2},"end":{"line":58,"column":2}}]},"6":{"line":66,"type":"if","locations":[{"start":{"line":66,"column":2},"end":{"line":66,"column":2}},{"start":{"line":66,"column":2},"end":{"line":66,"column":2}}]},"7":{"line":85,"type":"if","locations":[{"start":{"line":85,"column":6},"end":{"line":85,"column":6}},{"start":{"line":85,"column":6},"end":{"line":85,"column":6}}]},"8":{"line":100,"type":"if","locations":[{"start":{"line":100,"column":10},"end":{"line":100,"column":10}},{"start":{"line":100,"column":10},"end":{"line":100,"column":10}}]},"9":{"line":121,"type":"if","locations":[{"start":{"line":121,"column":6},"end":{"line":121,"column":6}},{"start":{"line":121,"column":6},"end":{"line":121,"column":6}}]},"10":{"line":129,"type":"if","locations":[{"start":{"line":129,"column":6},"end":{"line":129,"column":6}},{"start":{"line":129,"column":6},"end":{"line":129,"column":6}}]},"11":{"line":146,"type":"binary-expr","locations":[{"start":{"line":146,"column":6},"end":{"line":146,"column":22}},{"start":{"line":146,"column":26},"end":{"line":146,"column":58}}]}}},"/Users/nazli.gungor/Documents/refocus-ldk/bin/lensPrototype.js":{"path":"/Users/nazli.gungor/Documents/refocus-ldk/bin/lensPrototype.js","s":{"1":1,"2":1,"3":1,"4":1,"5":1,"6":1,"7":1,"8":1,"9":1,"10":0,"11":1,"12":1,"13":1},"b":{},"f":{"1":0,"2":1},"fnMap":{"1":{"name":"(anonymous_1)","line":18,"loc":{"start":{"line":18,"column":13},"end":{"line":18,"column":33}}},"2":{"name":"(anonymous_2)","line":22,"loc":{"start":{"line":22,"column":17},"end":{"line":22,"column":29}}}},"statementMap":{"1":{"start":{"line":8,"column":0},"end":{"line":8,"column":35}},"2":{"start":{"line":9,"column":0},"end":{"line":9,"column":29}},"3":{"start":{"line":10,"column":0},"end":{"line":10,"column":22}},"4":{"start":{"line":11,"column":0},"end":{"line":11,"column":26}},"5":{"start":{"line":12,"column":0},"end":{"line":12,"column":36}},"6":{"start":{"line":13,"column":0},"end":{"line":13,"column":18}},"7":{"start":{"line":15,"column":0},"end":{"line":15,"column":62}},"8":{"start":{"line":16,"column":0},"end":{"line":16,"column":29}},"9":{"start":{"line":18,"column":0},"end":{"line":20,"column":3}},"10":{"start":{"line":19,"column":2},"end":{"line":19,"column":50}},"11":{"start":{"line":22,"column":0},"end":{"line":25,"column":3}},"12":{"start":{"line":23,"column":2},"end":{"line":24,"column":43}},"13":{"start":{"line":27,"column":0},"end":{"line":27,"column":21}}},"branchMap":{}}}
```

Comment:
oops don't add coverage files to the repo

Reply:
Sorry I forgot about that folder! Will remove them